bookkeeper-server/src/main/java/org/apache/bookkeeper/net/NetUtils.java

```diff
@@ -85,4 +85,52 @@ public static String resolveNetworkLocation(DNSToSwitchMapping dnsResolver, Inet
         return rNames.get(0);
     }
 
+   public static String getLocalHost() throws UnknownHostException {
+        try {
+            String ip = InetAddress.getLocalHost().getHostAddress();
+            if (ip.equals("127.0.0.1")) {
+                return getAddress();
+            } else
+                return ip;
+        } catch (Throwable e) {
```

Comment:
Please catch the exact expected exception type

Reply:
This exception is superfluous and has been removed